provider/execution/execution.go

```diff
@@ -228,10 +228,9 @@ func (e TableExecutor) callTableResolve(ctx context.Context, client schema.Clien
 		if err := e.Table.Resolver(ctx, client, parent, res); err != nil {
 			if e.Table.IgnoreError != nil && e.Table.IgnoreError(err) {
 				client.Logger().Warn("ignored an error", "err", err, "table", e.Table.Name)
```

Comment:
nit: change this to debug level
```suggestion
				client.Logger().Debug("ignored an error", "err", err, "table", e.Table.Name)
```